contracts/LiquidPledgingBase.sol

```diff
@@ -122,11 +169,26 @@ contract LiquidPledgingBase is Owned {
 
     event GiverUpdated(uint64 indexed idGiver);
 
-    /// @notice Creates a new Delegate
-    function addDelegate(string name, string url, uint64 commitTime, ILiquidPledgingPlugin plugin) returns (uint64 idDelegate) { //TODO return index number
-        require(isValidPlugin(plugin));
+    /// @notice Creates a Delegate Admin with the `msg.sender` as the Admin addr
+    /// @param name The name used to identify the Delegate
+    /// @param url The link to the Delegate's profile often an IPFS hash
+    /// @param commitTime Sets the length of time in seconds the Delegate has to
+    ///   veto when the Delegate delegates to another Delegate and they pledge 
```

Comment:
This is used not only when "Delegate delegates to another Delegate", but when ever this delegate is in the delegationChain of a pledge.

when proposing a project, the max commitTime is used out of the pledge owner & all delegates in the chain.

Please update this to reflect the above comments

Reply:
"Sets the length of time in seconds that this delegate can be vetoed. Whenever this delegate is in a delegate chain the time allowed to veto any event must be greater than or equal to this time."